### PR TITLE
 Thank you in Indonesia Language

### PR DESCRIPTION
**Pull request by kol4k**
- [ ] Included a Preview link and screenshot showing after and before the changes.
- [ ] Included a description of the change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link](url)

- Fixes #<Issue Number>

### After:

- - - - - - - - - - - -

### CODE REVIEW

**Comment by kol4k** [comment]
@sarnava1 i've been squashing now,

**Comment by Ryan10145** [comment]
@kol4k Please provide a preview link

File: _data/students.yml [hunk 1]
```diff
@@ -138,3 +138,7 @@
 - name: coco
   github: cocox33x
   image: coco.png
+
+- name: Mochammad Rafi
```
Comment (jogendra) [inline _data/students.yml]: remove this.

File: _data/students.yml [hunk 2]
```diff
@@ -138,3 +138,7 @@
...
+  github: kol4k
```
Comment (jogendra) [inline _data/students.yml]: this one too

File: _data/students.yml [hunk 3]
```diff
@@ -138,3 +138,7 @@
...
+  image: rafi.jpeg
```
Comment (jogendra) [inline _data/students.yml]: one issue for one pr

**Pull request merged**